pkg/mtu/detect_linux.go

```diff
@@ -16,8 +16,15 @@ import (
 )
 
 const (
-	externalProbeIPv4 = "1.1.1.1"
-	externalProbeIPv6 = "2606:4700:4700::1111"
```

Comment:
Running the unit tests on GH actions were making this flake to happen frequently and I ended up changing as part of the PR to avoid introducing flakes with it.